frontend/src/components/util/Toggle/toggle.module.scss

```diff
@@ -0,0 +1,62 @@
+.container {
+  padding-left: 4rem;
+}
+
+.switch {
```

Comment:
let's make it a bit smaller as toggle button seems a big bigger than rest of the contents in the navbar